### DISCUSSION TITLE
Why was the app discontinued?

### DISCUSSION THREAD

**Discussion by DraconicNEO** [Shabinder/SpotiFlyer#3019] [Q&A] [answered]
Hello I wanted to ask how come the app was discontinued? I was under the impression that it was supposed to be rewritten but instead it was discontinued, being replaced by a new app which doesn't support desktop devices at all (at least as of writing), is google dependent, only supports m4a, and also unlike Spotiflyer doesn't appear to be open source (I couldn't find a source link, will edit this in the future if one is posted or I find a repository). It just seems like a big step backwards from what we already have here.

**Comment by Shabinder** [comment, marked as answer]
> Google dependent

It only uses analytics and crashlytics, will see about publishing a flavor without those if we a lot of demand in https://github.com/Shabinder/soundbound-extensions-lib/issues/1

> Only supports M4A

Spotiflyer always converted to mp3 from m4a, because it couldnt write metadata into m4a,

Soundbound can and hence doesnt convert files to mp3 anymore, any reason why mp3 is needed for your usecase.

Will consider adding back, but its just an extra overhead.

> doesn't appear to be open source

It is not at the time of writing, see : https://github.com/Shabinder/soundbound-extensions-lib/issues/1#issuecomment-1556258872


> It just seems like a big step backwards from what we already have here.

I can see your pov, but I will just request you to see how the app matures with time, in due time when all planned things are added, you will come to see why a core change to new app with different architecture was needed.
It is just first preview release.

**Reply by DraconicNEO** [reply]
>It only uses analytics and crashlytics, will see about publishing a flavor without those if we a lot of demand in https://github.com/Shabinder/soundbound-extensions-lib/issues/1

That's good so it should be easy to get working for people without Google play, problem is it's Play store exclusive so they don't have access to it. Yes I am aware that they can unofficially download it but that should not be encouraged as an alternative to officially distributing though methods that aren't dependent on the Play store.

>Soundbound can and hence doesnt convert files to mp3 anymore, any reason why mp3 is needed for your usecase.

Generally mp3 has better compression but also there are many Mp3 players and Multimedia devices which don't support m4a but do support mp3 so having the app output mp3 is more convenient than having to convert to different formats while also preserving metadata during the conversion.

>It is not at the time of writing, see : https://github.com/Shabinder/soundbound-extensions-lib/issues/1#issuecomment-1556258872

Unfortunate, hopefully this will change, I can see this being a deal-breaker for many, I can also see some feeling like it's a stab in the back for people who like to tinker and to [OSS](https://en.wikipedia.org/wiki/Open-source_software) in general. I draw no conclusions until you reach a final decision on that though.

>I can see your pov, but I will just request you to see how the app matures with time, in due time when all planned things are added, you will come to see why a core change to new app with different architecture was needed.
It is just first preview release.

I guess only time will tell. I hope the new one has a bright future I'm optimistic that it will assuming things go to plan.
